src/opnsense/service/templates/OPNsense/Netflow/netflow.conf

```diff
@@ -25,4 +25,6 @@ netflow_egress_only="{% if OPNsense.Netflow.capture.egress_only %}{% for interfa
 netflow_version="{%if OPNsense.Netflow.capture.version == 'v9' %}9{% else %}5{%endif%}"
 netflow_int_destination="127.0.0.1:2055"
 netflow_destinations="{{OPNsense.Netflow.capture.targets.replace(',', ' ')}}"
+netflow_active_timeout={{OPNsense.Netflow.activeTimeout}}
```

Comment:
@fbrendel as a safety precaution it might be better to add `|default()` as well, since netgraph will fail with pretty annoying errors if one of the timeouts is empty. (migration should prevent this, but I'm a bit worried about odd errors down the road if people restore a config without running migrations).

Reply:
Ok, commit is on the way.

Regarding the "annoying errors", wouldn't it be better then to have single ngctl commands instead of a batch in the rc.d script?
https://github.com/opnsense/core/blob/ef8cd8021dc91f5c05c3808d2e73e1de5c339498/src/etc/rc.d/netflow#L87-L95

Reply:
@fbrendel I haven't tried it myself, but it might be better for traceability indeed. Since this PR is rather small and focused, I rather keep the cleanup in a separate one. If you would like to test separate steps and create a PR afterwards, that would be nice.

Reply:
@AdSchellevis ok, then we'll see us in another PR :smiley:

Reply:
@fbrendel thanks!, I'll merge this one first.